res/css/views/rooms/_EventTile.pcss

```diff
@@ -648,6 +648,32 @@ $left-gutter: 64px;
             margin-top: 0;
             margin-bottom: 0;
         }
+
+        /* EventTile on every layout */
+        .mx_EventTile & {
+            /* Paragraph below ul or ol */
+            :is(ul, ol) + p {
+                margin-block: $spacing-16;
+            }
+        }
+
+        /* EventTile on compact modern layout only */
+        .mx_MatrixChat_useCompactLayout .mx_EventTile[data-layout="group"] & {
+            p,
+            ul,
+            ol,
+            dl,
+            blockquote,
+            pre,
+            table {
+                margin-bottom: $spacing-4; /* 1/4 of the non-compact margin-bottom */
+            }
+
+            /* Paragraph below ul or ol */
+            :is(ul, ol) + p {
+                margin-block: $spacing-4;
+            }
```

Comment:
again, how does this interact with the settings at lines 647-650?

Reply:
Ditto